### ISSUE TITLE
Errors in `motion_command.m` and `main.m` for `s1_octave_tutorial`

### ISSUE THREAD

**Issue by ritesh27gole** [conorhennessy/Computer-Vision-Playground#2]
In `s1_octave_tutorial`,

1. When I run `motion_command.m`, I get the following error: 
```cplusplus
>> motion_command
Not enough input arguments.

Error in motion_command (line 8)
x(1) = x(1) + u.t * cos(normalize_angle(x(3)) + u.r1);
```

2. I edited the `main.m` script since one for loop end wasn't defined.   
When I tried running, I got the following error:
```cplusplus
>> main
File: read_world.m Line: 19 Column: 11
Invalid use of operator.

Error in main (line 15)
landmarks = read_world('../data/world.dat');
```

**Comment by conorhennessy** [comment]
Hi there @ritesh27gole 👋🏻 

Just to confirm, these are errors you occur first when running `motion_command.m` and second for running `main.m` for `s1_octave_tutorial`?

Very good of you to report this issue if so, it has been a while since I have looked at this project and so I will investigate this in the next couple of days and I will get back to you as soon as I can.

**Comment by ritesh27gole** [comment]
> Hi there @ritesh27gole 👋🏻
> 
> Just to confirm, these are errors you occur first when running `motion_command.m` and second for running `main.m` for `s1_octave_tutorial`?
> 
> Very good of you to report this issue if so, it has been a while since I have looked at this project and so I will investigate this in the next couple of days and I will get back to you as soon as I can.

Hey @conorhennessy !
Yes I first saved both files, then ran `motion_command.m`, got the error corresponding to it, and then tried running `main.m` script anyways, and got the error corresponding to that.

Well thanks for getting back this early! I appreciate it as a beginner in the field of robotics and SLAM.

**Comment by conorhennessy** [comment]
First of all, my apologies for the delay in getting back to you.

I think you have not selected the right directory to run the `main` command. Or that you do not have the dataset required in `../data/`.

While my SLAM & VPR knowledge is well and truly rusty, I have not been able to replicate the issue you reported. 
I have just tested the current code with the existing datasets for `s1_octave_tutorial` and all has been able to run grand. As shown below here:
![an image showing s1_octave_tutorial running](https://user-images.githubusercontent.com/9257022/152655584-8afc093d-0279-484d-816d-27048372f6b9.png)

I would recommend cloning this repository. Then ensure you are running this code in Octave and make sure that you pick the right directory to run this in. For this first exercise (`s1_octave_tutorial`), I have selected the current directory as `C:/Users/Conor/Documents/Projects/SLAM-Course-Solutions/s1_octave_tutorial/octave` as `s1_octave_tutorial/octave` is the directory that contains `main.m` script. The script can then be run by issuing `main` in the command window with the right directory selected.

Further details & download for Octave are available [here](https://www.gnu.org/software/octave/download).
